Update README.md and Base64 encoding

ddarwin opened the pull request:
Update new README and correct Base64 encoded strings. The curl command downloaded and encoded the GitHub page for the gremlin-openshift4.cil file and not the contents of the file itself. I changed the curl command to pull down the RAW content, and updated the base64 encoded string in the MCO YAML as well as the gremlin-openshift4.cil.base64 file.

ddarwin committed:
Update README.md and Base64 encoding

Update new README and correct Base64 encoded strings

chuckersjp commented:
Interesting.

I agree that the URL should probably be updated, however the command you used to save the file appears to be MacOS specific and doesn't work on my RHEL box.

Can you include both lines?  The Linux version may need a redirect.

ddarwin committed:
Update Base64 encoding

ddarwin committed:
Update gremlin-openshift4.cil.b64

ddarwin committed:
Update README.md

ddarwin committed:
Update README.md

ddarwin commented:
Chuck, we had to make some additional changes to the repo, plus I made the documentation change that you recommended. The base64 encoded string was incomplete. Plus the Gremlin supplied file was actually the compiled SELinux Policy format (.cil) and not a SELinux Policy Package (.pp), so we changed that. And then I added the curl command, as you suggested, to include the RHEL command.

chuckersjp commented:
Thanks
